prefect_kubernetes/utilities.py

```diff
@@ -46,6 +48,38 @@ def enable_socket_keep_alive(client: ApiClient) -> None:
     ] = socket_options
 
 
+def hash_collection(collection) -> int:
```

Comment:
Would `prefect.utilities.hashing.hash_objects` work here?

Reply:
Didn't know we had that! I'll give it a try.